test/utils/ktesting/tcontext.go

```diff
@@ -241,7 +242,19 @@ func Init(tb TB, opts ...InitOption) TContext {
 	if c.PerTestOutput {
 		config := ktesting.NewConfig(
 			ktesting.AnyToString(func(v interface{}) string {
-				return format.Object(v, 1)
+				// For basic types where the string
+				// representation is "obvious" we use
+				// fmt.Sprintf because format.Object always
+				// adds a <"type"> prefix, which is too long
+				// for simple values.
+				switch v := v.(type) {
+				case int, int32, int64, uint, uint32, uint64, float32, float64, bool:
+					return fmt.Sprintf("%v", v)
+				case string:
+					return v
+				default:
+					return strings.TrimSpace(format.Object(v, 1))
+				}
```

Comment:
This seems to influence a lot of existing code. Would it make sense to do this in a separate PR?

Reply:
> This seems to influence a lot of existing code. 

Only tests, and only those using ktesting.

> Would it make sense to do this in a separate PR?

To simplify reverting? That can be done on a per-commit basis if needed, too.

I think a separate PR is overkill (each PR comes with its own costs), but I can create one if you prefer that.

Reply:
... in particular when we have many unrelated flakes which need retesting :cry: 

/retest

Reply:
@bart0sh: what do you think?

Can you review? It doesn't have to be @byako .

Reply:
sure,
/lgtm